callback_plugins/openshift_quick_installer.py

```diff
@@ -236,6 +236,63 @@ def v2_playbook_on_notify(self, res, handler):
         """
         self._display.display("skipping: no hosts matched", color=C.COLOR_SKIP, log_only=True)
 
+
+
+    ######################################################################
+    # So we can bubble up errors to the top
+    def v2_runner_on_failed(self, result, ignore_errors=False):
```

Comment:
Not a big deal but by default, this calls ```CallbackBase.runner_on_failed```

```python
    def v2_runner_on_failed(self, result, ignore_errors=False):
        host = result._host.get_name()
        self.runner_on_failed(host, result._result, ignore_errors)
```

which in turn is left to be implemented.

```python
    def runner_on_failed(self, host, res, ignore_errors=False):
        pass
```

In theory this would make sense to be put at ``runner_on_failed`` from the looks of the code. However, it doesn't really cause a problem.